lib/mods_display/fields/name.rb

```diff
@@ -30,6 +30,10 @@ def delimiter
       '<br />'
     end
 
+    def element_text(element)
```

Comment:
Can you add a comment that this overrides the parent class so that it makes text rather than HTML?

Reply:
Is the `xpath('.//text()')` necessary here?